lang/README.md

```diff
@@ -1 +1,16 @@
-A directory for language-specific parsers.
+# Language Specific Parsers
+
+This directory contains a set of language-specific parsers, that can be used to generate a class and file level dependency graph of your project. It outputs this dependency graph as a protobuff. For users of bfg, this dependency graph can be used to generate your BUILD files.
+
+We currently only support Java. However, we plan to add Scala support, and welcome contributions for other languages.
```

Comment:
I'd drop this sentence, it doesn't add information - the user can see which directories exist there